ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/config/JacksonConfiguration.java

```diff
@@ -18,6 +19,7 @@
     @Primary
     public ObjectMapper objectMapper() {
         return new ObjectMapper()
+            .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
```

Comment:
this is needed because ccd is returning `Yes` and `No` for their `YesOrNo` objects in a callback :(